src/main/resources/db.scripts.mvp/R__Session_Template_Data.sql

```diff
@@ -12,104 +12,151 @@ TRUNCATE TABLE session_to_permitted_location RESTART IDENTITY CASCADE;
+    ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'WEDNESDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-04',NULL,'FRIDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'SATURDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'SATURDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-04',NULL,'SUNDAY',true),
+    ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-04',NULL,'SUNDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'TUESDAY',true),
+    ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'TUESDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-09',NULL,'WEDNESDAY',true),
+    ('BLI_G2','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','16:00','2023-01-09',NULL,'FRIDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'SATURDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'SATURDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'14:00','15:00','2023-01-09',NULL,'SUNDAY',true),
+    ('BLI_G1','BLI','Visits Main Hall','SOCIAL',20,1,false,'15:30','16:30','2023-01-09',NULL,'SUNDAY',true),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'MONDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'TUESDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',0,2,false,'13:45','14:45','2023-01-23',NULL,'TUESDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',15,0,true,'18:00','18:45','2023-01-23',NULL,'TUESDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'WEDNESDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',0,2,false,'13:45','14:45','2023-01-23',NULL,'WEDNESDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'THURSDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',0,2,false,'13:45','14:45','2023-01-23',NULL,'THURSDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'FRIDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',0,2,false,'13:45','14:45','2023-01-23',NULL,'FRIDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'09:30','11:30','2023-01-23',NULL,'SATURDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','15:45','2023-01-23',NULL,'SATURDAY',false),
+    (NULL,'CFI','Visits Main Hall','SOCIAL',35,0,false,'13:45','14:45','2023-01-23',NULL,'SUNDAY',false),
```

Comment:
Finish time should be 15:45 not 14:45. 
https://dsdmoj.atlassian.net/browse/VB-1672
Sunday 13:45 - 15:45